docs/ref/settings.txt

```diff
@@ -739,6 +739,17 @@ The creation-order dependencies of the database. See the documentation
 on :ref:`controlling the creation order of test databases
 <topics-testing-creation-dependencies>` for details.
 
+.. setting:: TEST_MIGRATE
+
+``MIGRATE``
+^^^^^^^^^^^
+
+.. versionadded:: 3.1
+
+Default: ``True``
+
+When set to ``False``, migrations won't run when creating a test database.
```

Comment:
I think we should add a note to the `MIGRATION_MODULES` documentation about this setting.